lib/elixir/lib/enum.ex

```diff
@@ -1445,9 +1445,9 @@ defmodule Enum do
   ## Examples
 
       iex> Enum.reduce_while(1..100, 0, fn i, acc ->
-      ...>   if i <= 3, do: {:cont, acc + i}, else: {:halt, acc}
+      ...>   if i < 3, do: {:cont, acc + i}, else: {:halt, acc}
```

Comment:
@eksperimental I know this isn't a major change, but does this seem sufficient to you?